src/Controller/QuizTemplateController.php

```diff
@@ -39,14 +42,25 @@ public function createQuiz(Request $request)
 
     public function showQuizzes(Request $request, array $requestAttributes)
     {
-        $arguments['currentPage'] = (int)$request->getParameter('page');
-        $arguments['pages'] = $this->quizTemplateService->getQuizNumber($requestAttributes);
-        $arguments['username'] = $this->quizTemplateService->getName();
-        $arguments['quizzes'] = $this->quizTemplateService->getQuizzes($requestAttributes, $request->getParameter('page'));
-//        $arguments['quizUser'] = $this->quizTemplateService->getQuizUser();
-
-
-        return $this->renderer->renderView("admin-quizzes-listing.phtml", $arguments);
+        $resultsPerPage = 5;
```

Comment:
this should be a constant

Reply:
this will be received from a Combobox or something like this later

Reply:
yes, but for now it's ok for it to be a constant, and better. and you can make use of that constant in more places